api/templates/base.html

```diff
@@ -13,35 +13,26 @@
           crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
 
-    <!-- our custom css --> 
+    <!-- our custom css -->
     <link rel="stylesheet" href="{% static 'style/custom.css' %}">
-    
+
```

Comment:
The CO-CONNECT logo doesn't align with the links in the navbar....obviously not a huge deal int he grand scheme of things but alignment would make the navbar look better.